### ISSUE TITLE
Fix incorrect GPU decode on interlaced videos

### ISSUE THREAD

**Issue by willcrichton** [scanner-research/scanner#132]
See `tvnews/videos/MSNBCW_20170708_010000_The_Rachel_Maddow_Show.mp4` for an example.

![0](https://user-images.githubusercontent.com/663326/34969565-325243ee-fa24-11e7-9908-400fb186bd0f.jpg)

**Comment by Kuree** [comment]
It seems to be an odd interlace to me. Usually interlaced videos have horizontal lines, rather than vertical ones. I can be wrong.

**Comment by apoms** [comment]
Good point, maybe something other than interlacing is going on. Possibly a different YUV layout that we haven't accounted for?

**Comment by Kuree** [comment]
The entire `mp4` file only has one `avcC` box so I don't think it's going to switch `SPS` and `PPS` in the middle of presentation. That means the underlying video frames are not going to switch YUV layout, 4:2:0 in our case. 

It appears that half of the luma is missing and so does most of the chroma. I couldn't think of a plausible reason off my head now, but I will definitely going to take a closer look at it tomorrow. @willcrichton do you happen to know the frame number?

**Comment by willcrichton** [comment]
@Kuree that's frame 0
